.github/workflows/purge-cache.yml

```diff
@@ -0,0 +1,13 @@
+name: Purge Cache
+
+on:
+  workflow_call:
+
+jobs:
+  purge-jsdelivr-cache:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: gacts/purge-jsdelivr-cache@v1
+        with:
+          url: |
+            https://cdn.jsdelivr.net/npm/@trendyol/baklava@beta
```

Comment:
Purging that URL will not be enough. I asked this to jsDelivr before:

> Murat: Thank you for your response. Can we purge a prefix/alias like `/npm/@trendyol/baklava@beta/`? Or do we need to purge cache for every files specifically?
> Dmitriy: You would have to purge every file specifically. Most of our CDNs don't support prefix purging unfortunately.

So we need to purge every file url on `@beta` channel that has same name like `package.json`, `baklava.js`, `baklava-react.js`, `customElements.json`, all of the component entrance files (like `dist/components/button/bl-button.js`) and SVG files. :(

I think it'll be wise to find the URLs of the changed files only (especially for the SVGs)

Reply:
😞